app/models/candidate.rb

```diff
@@ -25,10 +25,16 @@ def prepare(user)
       self.stil_id = user.profile.stil_id
     end
   end
+
+  def owner?(user)
+    user.present? && user.profile == self.profile
+  end
+
   def editable?
     return (self.election.view_status == 2) || (self.post.elected_by == "Studierådet")
   end
+
   def p_url
-    Rails.application.routes.url_helpers.election_candidate_url(self.id,host: PUBLIC_URL)
+    Rails.application.routes.url_helpers.election_candidate_url(self.id, host: PUBLIC_URL)
```

Comment:
Will fix this and push with next commit (which will be tests for another controller).